wiki/Tournaments/Official_support/en.md

```diff
@@ -53,9 +53,7 @@ If your tournament satisfies the [eligibility criteria](#eligibility), you can m
     - For regional tournaments, the thread may be created in the appropriate language subforum.
   - Public Discord servers or any other off-site chatrooms used for the event.
   - Previous iterations of the same tournament series (only where applicable).
-- Separate lists for:
-  - The staff involved in running the tournament.
-  - The users who have registered to participate in the tournament (see [tournament screening](#tournament-screening)).
+- A separate list of the users who have registered to participate in the tournament (see [tournament screening](#tournament-screening)).
```

Comment:
```suggestion
- A list of the users who have registered to participate in the tournament (see [tournament screening](#tournament-screening)).
```

Reply:
what. if you want attachments then say so explicitly (it's not clear with current wording)

```suggestion
- A list of the users who have registered to participate in the tournament (see [tournament screening](#tournament-screening)), as a separate attachment.
```